features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store/site/themes/fancy/templates/subscription/subscription-list/js/subscription-list.js

```diff
@@ -445,20 +445,17 @@ var regenerate=function(appName,keyType,i,btn,div,clientId,clientSecret) {
                 if(scopeTxtBox.length==0){  // checking whether scopeTxtBox is already exist
 
                     //generate scopeTxtBox and append
-                   tokenTxtBox.before(
-                        function(){
-                            return  '<div class="row-fluid keys">'+
-                                    '<div class="span3">'+
-                                    '<b>Scope:</b>'+
-                                    '</div>'+
-                                    '<div class="span9">'+
-                                    '<div class="token">'+
-                                    '<span class="accessTokenScopeDisplayPro keyValues" id="prodAccessScope" data-value="'+generatedScopesNames+'">'+generatedScopesNames+'</span>'+
-                                    '</div>'+
-                                    '</div>'+
-                                    '</div>';
-                        }
-                    );
+                    var scopeTxtDiv=    '<div class="row-fluid keys">'+
```

Comment:
fixed in commit: https://github.com/wso2/carbon-apimgt/commit/d7e56b1d85254369d07aab9572f13d4c2120df97